test/sql/compaction/compaction_multiple_rename_column.test

```diff
@@ -0,0 +1,69 @@
+# name: test/sql/compaction/compaction_multiple_rename_column.test
+# description: test compaction with multiple column renames (regression test for WriteDroppedColumns bug)
+# group: [compaction]
+#
+# This test verifies that WriteDroppedColumns correctly sets end_snapshot only on the
+# currently active column entry (end_snapshot IS NULL), not on historical entries.
+#
+# Bug scenario: When renaming a column multiple times (A -> B -> A -> C), the last rename
+# would corrupt all historical entries for that column_id, making multiple entries visible
+# at intermediate snapshots. This caused "Column with name X already exists!" errors when
+# loading old schema versions (e.g., during compaction or time travel).
+
+require ducklake
+
+require parquet
+
+load __TEST_DIR__/compaction_multi_rename.db
+
+statement ok
+ATTACH 'ducklake:__TEST_DIR__/compaction_multi_rename_meta.db' AS ducklake (DATA_PATH '__TEST_DIR__/ducklake_compaction_multi_rename_files')
```

Comment:
Replace with, as this will allows us to run this test over multiple different configurations in our CI:

```
test-env DUCKLAKE_CONNECTION __TEST_DIR__/{UUID}.db

test-env DATA_PATH __TEST_DIR__

statement ok
ATTACH 'ducklake:${DUCKLAKE_CONNECTION}' AS ducklake (DATA_PATH '${DATA_PATH}/compaction_multiple_rename_column/', METADATA_CATALOG 'metadata');

statement ok
USE ducklake;

```